.wasser/config.yaml

```diff
@@ -55,10 +55,9 @@ routines:
         sudo systemctl start libvirtd
     - name: run microos image on vagrant
       command: |
-        cd aquarium/tests/vagrant
-        ./setup.sh foo 2>&1
-        cd setups/foo
-        vagrant up --no-destroy-on-error --no-parallel 2>&1
+        cd aquarium
+        ./tools/aqrdev create foo
```

Comment:
add `2>&1` at the end to see why it is failing...

Reply:
ack